From: Benjamin Drung <benjamin.drung@canonical.com>
Date: Wed, 30 Jul 2025 16:22:54 +0200
Subject: feat(debian.conf): default to initrd.img-${kernel} on Debian/Ubuntu

Extend the Debian/Ubuntu Dracut configuration to change the default
`initrdname` to `initrd.img-${kernel}`. This is done to keep the name
the same as initramfs-tools which has been the default for a long time.
This change will be carried for quite some time to ease the switch from
initramfs-tools to dracut.

Forwarded: https://github.com/dracut-ng/dracut-ng/pull/1484
---
 dracut.conf.d/debian/10-debian.conf | 1 +
 man/dracut.conf.5.adoc              | 3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/dracut.conf.d/debian/10-debian.conf b/dracut.conf.d/debian/10-debian.conf
index 0e043fe..c93cc4a 100644
--- a/dracut.conf.d/debian/10-debian.conf
+++ b/dracut.conf.d/debian/10-debian.conf
@@ -1,3 +1,4 @@
 # Debian/Ubuntu specific Dracut configuration
 hostonly="yes"
 hostonly_cmdline="no"
+initrdname="initrd.img-${kernel}"
diff --git a/man/dracut.conf.5.adoc b/man/dracut.conf.5.adoc
index 9290758..77d5dca 100644
--- a/man/dracut.conf.5.adoc
+++ b/man/dracut.conf.5.adoc
@@ -370,7 +370,8 @@ The initrdname configuration option is required to match the
 _initr*$\{kernel\}*_ file pattern and only one file with this pattern should
 exists in the directory where initramfs is loaded from.
 +
-Defaults to _initramfs-$\{kernel\}.img_.
+Defaults to _initramfs-$\{kernel\}.img_. Debian/Ubuntu based systems normally
+ship a configuration file to change the defaults to _initrd.img-$\{kernel\}_.
 
 == Files
 _/etc/dracut.conf_::
